IPython/html/static/notebook/js/main.js

```diff
@@ -76,7 +76,8 @@ require([
         events: events, 
         keyboard_manager: keyboard_manager});
     var contents = new contents.Contents($.extend({
-        events: events, config:config_section},
+          common_config: common_config
```

Comment:
did you want to make these contents constructors only take base_url, common_config like above?